data/human/human missions.txt

```diff
@@ -6893,3 +6893,51 @@ mission "Returning to Paradise"
+			`	For the event, the transport company decided to have a public poll to name the ship, and while initial suggestions were casual names such as "Stark Lady" or "Timelight", some of the most popular ones are actually a collection of joke names - ones you would imagine a toddler would have made up. Apparently, the names have gotten enough attention and enjoyment out of pilots that the company decided to take the top five most popular results and hold a vote on them.`
+			`	Would you like to vote on the name as well?`
+			choice
+				`	(Yes.)`
+				`	(No.)`
+					decline
+			`	You navigate the article until you are directed to the poll itself, presenting you with the top five options.`
+			choice
+				`	Star Empress`
+				`	Tycho's Legacy`
+				`	ShippyMcShipFace`
+				`	It's Dark Here`
+				`	What Asteroid`
+			action
+				event "hogshead poll" 7
+			`	After you made your selection, a "thank you" message appears, telling you that the result of the poll will be broadcast on the news once the voting period has finished. Afterwards, you go back about your business.`
+				decline
+
+event "hogshead poll"
+
+mission "Hogshead's Dilemma II"
+	invisible
+	source
+		near Sol 1 10
+	to offer
+		has "Hogshead's Dilemma: offered"
+		has "event: hogshead poll"
+	on offer
+		clear "hogshead poll"
+		conversation
+			`As you are walking around the spaceport, you see that more than a few pilots are watching the news on nearby displays. There are mixed emotions amongst the crowd - some pilots are having a good laugh, while others seem disappointed, or even confused. You aren't sure what this is about until, lo and behold, the result of that Hogshead naming poll from a while ago is clearly shown on a display - ShippyMcShipFace. The sheer ridiculousness of the situation is capturing the attention of almost anyone who catches a glimpse of the monitor. You can't say it's all that much worse than some of the names you've heard pirates give their ships before, but it still seems silly that the transport company is actually going through with the name.`
+			`	You recall that, back in their day, Hogsheads were sort of a business fad thanks to a ton of discussion and nicknames based on the shape of their engine pods. In turn, this gave them a brief but strong popularity amongst organizations looking to grab the average passenger's attention. In the end, the transport company responsible for the naming poll got what they wanted, and a little bit of the Hogshead's spirit has been brought back. Perhaps the name "ShippyMcShipFace" is fitting after all.`
```

Comment:
Are you sure about First Last **recalling** this information, which is being introduced to the player for the first time? Would the character have any reason to know that?

Reply:
If the Hogshead was such a fad/meme back in the day, it makes sense that it would be on historical channels or documents that the player gets involved in as an aspiring pilot. Same with if a pilot today learned of the A-10 or something similar. The player is also said to already have knowledge of the Quarg from media, etc., prior to meeting them, so I feel like this is within reason.